mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -0,0 +1,247 @@
+        :param issue_num: The specific issue number we want to label
+        :param labels: The labels which we want to remove
+        :return Response denoting success or failure for logging purposes
+        """
+        labels = self.format_labels(labels)
+        issue_labels_url = 'https://api.github.com/repos/{repo}/issues/{id}/labels/' \
+            .format(repo=self.repo, id=issue_num)
+
+        for label in labels:
+            delete_label_url = issue_labels_url + label
+            response = requests.delete(delete_label_url, auth=self.auth)
+            if response.status_code == 200:
+                logging.info('Successfully removed label to {}: {}.'.format(str(issue_num), str(label)))
+            else:
+                logging.error('Could not remove the label to {}: {}.'.format(str(issue_num), str(label)))
+                logging.error(response.json())
+                return "Failed to remove labels"
+        return "Removed labels successfully"
+
+    def update_github_labels(self, issue_num, labels):
+        """
+        This method is to update a list of labels to one issue.
+        It checks whether labels exist in the repo, and updates existing labels to the issue
+        :param issue_num: The specific issue number we want to label
+        :param labels: The labels which we want to remove
+        :return Response denoting success or failure for logging purposes
+        """
+        labels = self.format_labels(labels)
+        issue_labels_url = 'https://api.github.com/repos/{repo}/issues/{id}/labels' \
+            .format(repo=self.repo, id=issue_num)
+
+        response = requests.put(issue_labels_url, data=json.dumps(labels), auth=self.auth)
+        if response.status_code == 200:
+            logging.info('Successfully updated labels to {}: {}.'.format(str(issue_num), str(labels)))
+            return "Updated labels successfully"
+        else:
+            logging.error("Could not update the labels")
+            logging.error(response.json())
+            return "Failed to update labels"
+
+    def add_comment(self, issue_num, message):
+        """
+        This method will trigger a comment to an issue by the label bot
+        :param issue_num: The issue we want to comment
+        :param message: The comment message we want to send
+        :return Response denoting success or failure for logging purposes
+        """
+        send_msg = {"body": message}
+        issue_comments_url = 'https://api.github.com/repos/{repo}/issues/{id}/comments' \
+            .format(repo=self.repo, id=issue_num)
+
+        response = requests.post(issue_comments_url, data=json.dumps(send_msg), auth=self.auth)
+        if response.status_code == 201:
+            logging.info('Successfully commented')
+            return "Commented successfully"
+        else:
+            logging.error("Could not comment")
+            return "Failed to comment"
```

Comment:
Why are we returning response?
We should either log or return values - not both. In this case as we are logging, returning is not needed. Please remove.